Islandora-CLAW/islandora
# Initial implementation of a REST Export serializer for IIIF Manifests

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/1263

# What does this Pull Request do?

Create a plugin for generating an IIIF manifest from views results.

# What's new?
Adds a submodule `islandora_iiif` that has a Views style plugin for a serializer that prints IIIF URLs for images displayed in a View.

# How should this be tested?

* Enable `islandora_iiif` submodule
* Add your IIIF Server URL (the same one you'd have configured for OSD) at `/admin/config/islandora/iiif`
* Import this "test" // "demo" view: https://d.pr/f/DV5GID
* Access a node that has an image referencing the node via `field_media_of` e.g. /node/1
* Add "iiif-manifest" to the URL e.g. /node/1/iiif-manifest
* You should see the JSON of every image for the respective node

# Additional Notes:
For the Paged Content Sprint

# Interested parties
@Islandora-CLAW/committers
